packages/esm-active-visits-app/package.json

```diff
@@ -13,7 +13,10 @@
     "debug": "npm run serve",
     "build": "webpack --mode production",
     "analyze": "webpack --mode=production --env.analyze=true",
-    "lint": "eslint src --ext ts,tsx",
+    "lint": "cross-env TIMING=1 eslint src --ext ts,tsx",
+    "test": "cross-env TZ=UTC jest --config jest.config.js --verbose false --passWithNoTests",
```

Comment:
Can't we just use `../../jest.config.js`? This actually creates duplication and becomes tiring in a breaking change.
Thanks!

Reply:
I know we are using `require('../../jest.config.js')` but we can't we use it directly from the root?

Reply:
Thanks for the spot. Makes sense!

Reply:
Update: Per our discussion from earlier today, we've chosen to go with multiple jest configs because of the performance issues when using a single jest config.